packages/caver-rpc/src/governance.js

```diff
@@ -161,13 +161,37 @@ const Governance = function Governance(...args) {
          * const result = await caver.rpc.governance.getChainConfig()
          *
          * @param {function} [callback] Optional callback, returns an error object as the first parameter and the result as the second.
-         * @return {Promise<object>} The initial chain configuration
+         * @return {Promise<object>} The latest chain configuration
          */
         new Method({
             name: 'getChainConfig',
             call: 'governance_chainConfig',
             params: 0,
         }),
+        /**
+         * Returns the chain configuration at specific block.
+         * If the parameter is not set, it returns the chain configuration at the latest block.
+         * To see the current information, please use {@link itemsAt}.
+         *
+         * @memberof Governance
+         * @method getChainConfigAt
```

Comment:
@nohkwak Isn't it `ChainConfigAt`? cause the rpc call method name is `governance_chainConfigAt`, right?

Reply:
Oh, right.. Thanks!